src/applications/vaos/containers/PreferredDatePage.jsx

```diff
@@ -28,7 +28,21 @@ const uiSchema = {
   preferredDate: {
     'ui:title': 'What is the earliest date you’d like to be seen?',
     'ui:widget': 'date',
-    'ui:validations': [validateCurrentOrFutureDate],
+    'ui:validations': [
+      (errors, preferredDate) => {
+        const maxDate = moment()
+          .add(13, 'months')
+          .format('YYYY-MM-DD');
```

Comment:
Instead of doing the `format` call here and below, you could pass 'day' to `isAfter` and `isBefore` (e.g. `moment(preferredDate).isAfter(maxDate, 'day')`. That limits the comparison to just the day (excluding the time of day), which is effectively what you're doing with the `format` call (see https://momentjs.com/docs/#/query/is-after/).

What you have now works perfectly fine, but it's a roundabout way of making sure we're comparing just dates and not dates and times. Using the second param of isAfter/isBefore is more clear about the intent of the code, I think.

Reply:
Thanks for the link. Had no idea about existence of second param. It does make comparison more concise.